README.md

```diff
@@ -123,16 +124,21 @@ Install all the dependencies from your Gemfile using Bundler:
 
 	$ bundle install
 
-Run any database migrations:
+Set up your database and seed sample data (including admin users to log in!):
 
 	$ bundle exec rake db:migrate
+	$ bundle exec rake db:seed
```

Comment:
These two could be replaced with `bundle exec rake db:setup`; http://stackoverflow.com/a/10302357/6046659